src/main/java/seedu/address/model/country/CountryCodeVerifier.java

```diff
@@ -3,11 +3,12 @@
 import java.util.Locale;
 
 /**
- * Class that is responsible for verification of ISO3166 country-codes.
+ * The class that is responsible for verification of ISO3166 country-codes.
  */
 public class CountryCodeVerifier {
 
-    public static final String MESSAGE_CONSTRAINTS = "Country code must be a valid 2-letter ISO3166 country code";
+    public static final String MESSAGE_CONSTRAINTS = "Country code must be a valid "
+            + "2-letter ISO3166 country code";
```

Comment:
Not sure why this message was broken up into two lines

Reply:
It will exceed the 120 char limit if its on the same line